third_party/libtock-drivers/src/nfc.rs

```diff
@@ -63,34 +65,21 @@ impl NfcTag {
 
     /// Configure the tag type command.
     pub fn configure(tag_type: u8) -> bool {
-        let result_code =
-            syscalls::command(DRIVER_NUMBER, command_nr::CONFIGURE, tag_type as usize, 0);
-        if result_code.is_err() {
-            return false;
-        }
-
-        true
+        syscalls::command(DRIVER_NUMBER, command_nr::CONFIGURE, tag_type as usize, 0).is_ok()
     }
 
-    /// Set the maximum frame delay value to support
-    /// transmission with the reader.
+    /// Set the maximum frame delay value to support transmission with the reader.
     pub fn set_framedelaymax(delay: u32) -> bool {
-        let result_code =
-            syscalls::command(DRIVER_NUMBER, command_nr::FRAMEDELAYMAX, delay as usize, 0);
-        if result_code.is_err() {
-            return false;
-        }
-
-        true
+        syscalls::command(DRIVER_NUMBER, command_nr::FRAMEDELAYMAX, delay as usize, 0).is_ok()
     }
 
     /// 1. Share with the driver a buffer.
     /// 2. Subscribe to having a successful receive callback.
     /// 3. Issue the request for reception.
-    pub fn receive(buf: &mut [u8]) -> bool {
+    pub fn receive(buf: &mut [u8; 256]) -> TransOrRecvStatus {
```

Comment:
If this status wouldn't contain the success, it would instead return
`Result<(), TransOrRecvError>`
or
`Result<(), WhateverLibtockError>`
and you'd have access to `Result` magic like the `?` operator later.